doc/source/data/dataset-tensor-support.rst

```diff
@@ -6,15 +6,15 @@ ML Tensor Support
 Tensor (multi-dimensional array) data is ubiquitous in ML workloads. However, popular data formats such as Pandas, Parquet, and Arrow don't natively support tensor data types. To bridge this gap, Datasets provides a unified tensor data type that can be used to represent and store tensor data:
 
 * For Pandas, Datasets will transparently convert ``List[np.ndarray]`` columns to and from the :class:`TensorDtype <ray.data.extensions.tensor_extension.TensorDtype>` extension type.
-* For Parquet, the Datasets Arrow extension :class:`ArrowTensorType <ray.data.extensions.tensor_extension.ArrowTensorType>` allows Tensors to be loaded and stored in Parquet format.
+* For Parquet, the Datasets Arrow extension :class:`ArrowTensorType <ray.data.extensions.tensor_extension.ArrowTensorType>` allows tensors to be loaded from and stored in Parquet format.
 * In addition, single-column Tensor datasets can be created from NumPy (.npy) files.
```

Comment:
In the below example, a "multi-column tensor dataset" means all columns are tensor-typed. I was going to add a line about a "tensor dataset" concept, but I'm going to hold off on that since (1) that's a larger refactoring of this tensor support doc, and (2) I don't know if we're internally aligned on the messaging there yet.